charts/lnd/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
I think this should be expended:

how to get the TLS, macaroon, execute a lncli on the pod, etc.